include/mxnet/c_predict_api.h

```diff
@@ -42,7 +42,9 @@ extern "C" {
 #endif
 
 /*! \brief manually define unsigned int */
-typedef unsigned int mx_uint;
+typedef uint32_t mx_uint;
```

Comment:
still feel this typedef is redundant and decrease readability. The fact the same data types are defined in two different headers might cause include error. But it's okay to change this in a different PR.